src/components/ui/ListItem/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { ReactNode } from "react";
+import {
+  ListItem as RNListItem,
+  ListItemProps as RNListItemProps,
+} from "react-native-elements";
+import { Icon } from "..";
+import { colors } from "@app/utils";
+
+interface ListItemProps extends RNListItemProps {
+  id: string;
+  title: string;
+  otherInformation: ReactNode | string; //for other elements to display below the title e.g. description and pending bar
+  iconName?: string;
+  iconColor?: string;
+  iconFamily?: string;
+  showNav: boolean; //show chevron right navigation icon
+  onClick?: () => void;
+}
+
+const ListItem: React.FC<ListItemProps> = (props) => {
```

Comment:
It's better to do it like this for default values. Also, make the ones with default values as optional props

```
ListItem.defaultProps = {
  iconName: "md-basket",
  showNav: true,
  iconColor: "grey",
  iconFamily: "ionicon"
}
```